MAINTAIN.md

```diff
@@ -69,7 +69,7 @@ remove existing features, or refactor parts of the code that would change
 user's workflow. In these cases, a deprecation policy is needed to properly
 inform users of the change.
 
-In general, when a feature is slated to be removed it should:
```

Comment:
This was me adding some intentional wiggle room into the policy, but obviously being more explicit is better.